packages/expo-router/src/toolbar/elements.tsx

```diff
@@ -238,9 +260,9 @@ export interface ToolbarButtonProps {
    * import { useImage } from 'expo-image';
    * import { Toolbar } from 'expo-router/unstable-toolbar';
    *
-   * const customIcon = useImage('https://example.com/icon.png', {
-   *   maxWidth: 44,
-   *   maxHeight: 44,
+   * const customIcon = useImage('https://picsum.photos/128', {
```

Comment:
Maybe something a bit more durable like github.com/expo.png or more practical like svg:

```suggestion
   * const customIcon = useImage('https://simpleicons.org/icons/expo.svg', {
```